sharding-core/sharding-core-common/src/main/java/org/apache/shardingsphere/core/rule/ShardingRule.java

```diff
@@ -405,7 +405,17 @@ public DataNode getDataNode(final String dataSourceName, final String logicTable
         }
         throw new ShardingConfigurationException("Cannot find actual data node for data source name: '%s' and logic table name: '%s'", dataSourceName, logicTableName);
     }
-    
+
+    /**
+     * Judge if defaultDataSourceName exists.
```

Comment:
`defaultDataSourceName` are 4 words, please split it

Reply:
ok